src/inject.js

```diff
@@ -217,28 +244,23 @@ export const injectAndGetClassName = (
     styleDefinitions /* : MaybeSheetDefinition[] */,
     selectorHandlers /* : SelectorHandler[] */
 ) /* : string */ => {
-    styleDefinitions = flattenDeep(styleDefinitions);
+    const processedStyleDefinitions /* ProcessedStyleDefinitions */ = {
```

Comment:
nit: I think you need a `:` before `ProcessedStyleDefinitions` here in order to make flow actually notice this.